content/projects/data-science-specific/data-wrangling/_index.md

```diff
@@ -79,25 +80,24 @@ df.apply(get_percentage, axis = 1) #axis=1 applies the function to all columns
 ├──requirements.txt
 └──.gitignore 
 ```
-2. Import the dataset [personality_scores.csv](personality_scores.csv). This data should be named `personality_df` Examine the DataFrame for duplicates (based on ID), and drop any duplicates that exist. Use an assert statement to check that the DataFrame is the length of the unique entries of the original data frame. This new DataFrame should be named `personality_score_df`. If there are empty columns, please ensure you remove them.
+2. Import the dataset [personality_scores.csv](personality_scores.csv) and load it into a DataFrame. Examine the DataFrame for duplicates (based on ID), and drop any duplicates that exist. Let's imagine that the data is ordered chronologically, and we'd like to keep the data related to the most recent time an applicant did the test. From your recent project on [assertive programing tricks](projects/data-science-specific/assertive-programming-tricks-for-pandas/_index.md) use the methods you practiced to check that the updated DataFrame has the dimensions you expect it to have. Perform some final cleaning steps by removing empty columns and cleaning up column names. Store this updated version of the DataFrame in a variable called: `personality_score_df`.
```

Comment:
Why do you say "import" the dataset? is that standard terminology in data science? importing means something else usually. It might be good to say "Start off by loading the data in personality_scores.csv into a dataframe" instead just so it is precise. 

Why "let's imagine?" 
Can't we tell people to just sort it by date? And then deduplicate? Or is there no date column? 
Even if there is no date column I wouldn't say "let's imagine" I would say, the data is ordered chronologically. 

Please use contentlink shortcodes for links to other projects. I dont think that link would work on the live site at all.

I'm not sure if people will know what it means to clean up column names. Is there something we have taught them before that would help here? Or do they need an example? 

Lastly, I do think that giving learners such long and detailed paragraphs will probably lead to confusion. A lot of folks are not first language English speakers and so it is espectially useful to try to write things in a way that is extra super clear.

Reply:
Hey Sheen, import seems to be used interchangeably with load as far as I've seen, it was not something from the original instructions I really feel like needs overhauling...

Sure we can take out my lyrical word choices. lol
There is no date column and I don't want to add more complexity to the data.

I think the example of what "the dataframe should look something like this" in the instructions should be a good enough clue for what cleaning the columns up would be? Else I'll put it back the way Aminat had it :)